onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1541,6 +1584,439 @@ with the exception that numpy default keepdims to False instead of True.)DOC")
           "Keep the reduced dimension or not, default 1 mean keep reduced dimension.",
           AttributeProto::INT);
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(QLinearAdd)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .SetDoc(R"DOC(Performs element-wise binary quantized addition (with Numpy-style broadcasting support).
+This operator supports **multidirectional (i.e., Numpy-style) broadcasting**.
+```
+C = (A_scale * (A - A_zero_point) + B_scale * (B - B_zero_point))/C_scale + C_zero_point
+```
+)DOC")
+      .Input(0, "A", "First operand.", "T")
```

Comment:
For code size, don't QLinearAdd and QLinearMul have the same schema parameters? Can this be simplified to have a QMathDocGenerator as done in the base Add/Mul schema?